crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -172,6 +175,7 @@ impl InboundGroupSession {
         session_key: &SessionKey,
         encryption_algorithm: EventEncryptionAlgorithm,
         history_visibility: Option<HistoryVisibility>,
+        authenticated: bool,
```

Comment:
Why is this useful? This constructor, and `SessionKey` are only used/available if we received the key as a `m.room.key` event.

This is why the `imported` flag is further bellow set to `false` and why the `authenticated` flag should always be `true`.

Is this ever called with the argument being `false`?